Orange/util.py

```diff
@@ -1,21 +1,32 @@
 """Various small utilities that might be useful everywhere"""
+import logging
 import os
 import inspect
 from enum import Enum as _Enum
 from functools import wraps, partial
 from operator import attrgetter
 from itertools import chain, count, repeat
 
-from collections import OrderedDict
+from collections import OrderedDict, namedtuple
 import warnings
 
+try:
+    from PyQt5.QtCore import pyqtWrapperType
```

Comment:
While we (probably?) don't support PyQt4 anymore it would still be better to use `AnyQt` for imports...